k8s/base/1_configmap.yaml

```diff
@@ -176,7 +164,12 @@ data:
           - otlp
       telemetry:
         metrics:
-          address: ${env:MY_POD_IP}:8888
+          readers:
+          - periodic:
+              exporter:
+                otlp:
+                  protocol: grpc/protobuf
+                  endpoint: ${env:MY_POD_IP}:14317
```

Comment:
The conventional port to use for grpc is 4317. 

Is this a typo ? 

Also, the protocol string for grpc is I think `grpc`. For http it is `http/protobuf`. \
Just confirming if this is intentional ?

Reply:
The string is taken frmo here: https://opentelemetry.io/docs/collector/internal-telemetry/#configure-internal-metrics

Reply:
Thanks for the explanation. It makes sense.

I see, its different for internal metrics.